### PR TITLE
Created portfolio "about me" section

### PR DESCRIPTION
**Pull request by jorgeabrahan**
Created about me section HTML, about me styling, and add about me decorators.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -188,3 +201,199 @@ Headline
...
+  display: grid;
+  gap: 1.5rem;
+  margin-bottom: 1.5rem;
+}
+
+.works__card {
+  border: 3px solid #d4dcd8;
+  border-radius: var(--border-radius);
+  overflow: hidden;
+}
```
Comment (levy002) [inline css/styles.css]: - [x] Kindly make sure the contents of each individual project are aligned using flexbox in order to meet the project requirements. :wink:

File: css/styles.css [hunk 2]
```diff
@@ -188,3 +201,199 @@ Headline
...
+
+.card__technologies {
+  color: #3a4a42;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+  padding-left: 0;
+  list-style-type: none;
+  margin-block: 1rem;
+}
```
Comment (levy002) [inline css/styles.css]: - [x] Kindly make sure the project stack list is aligned in the center of the card container in order to meet the provided Figma design.

    - ### YOUR DESIGN
      image

    - ### BETTER DESIGN
    image

File: css/styles.css [hunk 3]
```diff
@@ -188,3 +201,199 @@ Headline
...
+  margin-bottom: 1.5rem;
+}
+
+.card__list {
+  padding-left: 0;
+  list-style-type: none;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+}
```
Comment (levy002) [inline css/styles.css]: - [x] Kindly make sure the LANGUAGES, SKILLS, AND FRAMEWORKS list is aligned in the center of the card container in order to meet the provided Figma design.

    - ### YOUR DESIGN
      image

    - ### BETTER DESIGN
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @jorgeabrahan   , :wave:
### I'm Levy, your PR reviewer

Excellent work :sunglasses: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
### Hi @jorgeabrahan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**